src/app/legacy/containers/StoryPromo/utilities/index.js

```diff
@@ -4,11 +4,15 @@ import {
   MEDIA_ASSET_PAGE,
 } from '#app/routes/utils/pageTypes';
 
+// MAP is now either a Media Asset Page or a Media Article Page
 export const isMap = item => {
   const isCpsTypeMap = pathOr(null, ['cpsType'], item) === MEDIA_ASSET_PAGE;
   const hasMedia = pathOr(false, ['media'], item);
```

Comment:
Ah yes I see. Means there's probably some duplication somewhere, but that's OK, because we know these pages will be going away early next year 🙌 

Thanks for explaining!